docs/source/visualization/TimeSeriesAnomalies.rst

```diff
@@ -621,18 +621,18 @@ other suspicious activity from other datasources.
 
 
 Time Series Anomalies Visualization
-===================================
+-----------------------------------
 
 Time series anomalies once discovered, you can visualize with line chart
 type to display outliers. Below we will see 2 types to visualize using msticpy function
 ``display_timeseries_anomalies()`` via Bokeh library as well as using
 built-in KQL ``render``.
 
 Using Bokeh Visualization Library
----------------------------------
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 Documentation for display_timeseries_anomalies
-----------------------------------------------
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

```

Comment:
sometime you replace ---- with ^^^ and sometimes with ~~~. Not sure if this is deliberate or not.